tsconfig.json

```diff
@@ -11,5 +11,15 @@
         "strictNullChecks": false,
         "noUnusedLocals": true
     },
-    "include": ["src/**/*.ts", "src/**/*.tsx", ".eslintrc.js", "src/**/*.js"]
+    "include": [
+        ".eslintrc.js",
+        "src/**/*.ts",
+        "src/**/*.tsx",
+        "src/**/*.js",
+        "deploy/**/*",
+        "packages/**/*",
```

Comment:
Stuff under `packages/` probably shouldn't be included here; tsc won't be able to build it correctly if they have separate `package.json`s with different dependencies (that's where the current PR build failures are mostly coming from).

Instead, this should probably follow `accessibility-insights-service`'s example, where `/package/*/package.json` specifies a `lint:check` script in terms of `../../.eslintrc.js` (like report-e2e-tests does already), and then use `lerna run lint:check` to run in all packages (maybe via a `lint:check:all` command at the root)